File: java/jimmer-cloud/book-service/src/main/java/org/babyfish/jimmer/example/cloud/book/BookServiceApp.java
package org.babyfish.jimmer.example.cloud.book;

import org.babyfish.jimmer.client.EnableImplicitApi;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableImplicitApi
@SpringBootApplication
public class BookServiceApp {

    public static void main(String[] args) {
        SpringApplication.run(BookServiceApp.class, args);
    }
}

File: kotlin/jimmer-cloud-kt/book-service/src/main/kotlin/org/babyfish/jimmer/example/cloud/kt/book/BookServiceApp.kt
package org.babyfish.jimmer.example.cloud.kt.book

import org.babyfish.jimmer.client.EnableImplicitApi
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@EnableImplicitApi
@SpringBootApplication
class BookServiceApp

fun main(args: Array<String>) {
    runApplication<BookServiceApp>(*args)
}